docs/sources/datasources/tempo.md

```diff
@@ -57,6 +57,12 @@ This is a configuration for the beta Node Graph visualization. The Node Graph is
 
 -- **Enable Node Graph -** Enables the Node Graph visualization.
 
+### Loki Search
+
+This is a configuration for the Loki search query type.
+
+-- **Data source -** Loki instance used when searching for traces. The Loki instance must have derived fields configured.
```

Comment:
```suggestion
-- **Data source -** The Loki instance in which you want to search traces. You must configure derived fields in the Loki instance.
```

Reply:
Committing in the UI isn't working right now, so I'll manually add in those changes